examples/apps/cra-demo/src/App.js

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The build system might be upset without the "and contributors" bit in the copyright header

Reply:
yeah, i copied over from an old file, must be an updated script